initfiles/componentfiles/configschema/xsd/environment.xsd

```diff
@@ -20,33 +20,33 @@
     xmlns:xs="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified" attributeFormDefault="unqualified"
     xmlns:hpcc="something hpcc.xsd">
     <xs:include schemaLocation="types.xsd"/>
-    <xs:include schemaLocation="hardware.xsd"/>
-    <xs:include schemaLocation="esp.xsd"/>
-    <xs:include schemaLocation="dali.xsd"/>
+    <!--<xs:include schemaLocation="hardware.xsd"/>-->
+    <!--<xs:include schemaLocation="esp.xsd"/>-->
+    <!--<xs:include schemaLocation="dali.xsd"/>-->
     <!--xs:include schemaLocation="esp_service_wsecl2.xsd"/-->
-    <xs:include schemaLocation="secmgr_singleuser.xsd"/>
-    <xs:include schemaLocation="esp_service_smc.xsd"/>
-    <xs:include schemaLocation="dafilesrv.xsd"/>
+    <!--<xs:include schemaLocation="secmgr_singleuser.xsd"/>-->
+    <!--<xs:include schemaLocation="esp_service_smc.xsd"/>-->
+    <!--<xs:include schemaLocation="dafilesrv.xsd"/>-->
```

Comment:
Why not remove those 3 lines?

Reply:
All commented lines have been removed since the new mechanism does not require explicit include statements